### PR TITLE
fix(susemanager-sls): Oracle Linux Bootstrap Issue

### PR DESCRIPTION
**Pull request by wombelix**
## What does this PR change?

Fix Error 'no match for argument: salt-minion' when bootstrapping Oracle Linux Clients due to missing oracle entry in 'RedHat OS Family'.

## GUI diff

No difference.

Before:

After:

- [X] **DONE**

## Documentation
- No documentation needed: Only Bugfix

- [x] **DONE**

## Test coverage
- No tests: Nothing changed on existing tests / no new tests added.

- [x] **DONE**

## Links

Fixes: https://github.com/uyuni-project/uyuni/issues/5061
Issue introduced in PR: https://github.com/uyuni-project/uyuni/pull/4249

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: susemanager-utils/susemanager-sls/susemanager-sls.changes [hunk 1]
```diff
@@ -1,3 +1,5 @@
+- Fix Oracle Linux Bootstrap issue 'No match for argument: salt-minion'
```
Comment (vzhestkov) [inline susemanager-utils/susemanager-sls/susemanager-sls.changes:1]: Not really clear enough, maybe reword it a bit:
```suggestion
- Fix bootstrap repository path resolution for Oracle Linux
```

**Review by vzhestkov — APPROVED** [review]
@wombelix thanks for the PR, just only one nitpicky suggestion for the changelog entry.
It's better to reword it to make more clear for the users.

**Comment by juliogonzalez** [comment]
@wombelix, if you want this into Uyuni 2022.04, I need you to close this PR and greate it against the branch `Uyuni-2022.04`, as I already branched in preparation for the release on Monday.

Otherwise it will be part of Uyuni 2022.05.

**Comment by wombelix** [comment]
@vzhestkov thanks for the suggestion, included.
@juliogonzalez I created the new PR https://github.com/uyuni-project/uyuni/pull/5260 which replace this one so you can include it in Uyunu-2022.04
